src/components/Button/Button.tsx

```diff
@@ -25,14 +25,16 @@ export function Button(props: ButtonProps) {
                           active:bg-neutral/[.3]                                                  
                           disabled:text-f-disabled-1 disabled:bg-c-disabled-2 disabled:border-c-disabled-1`;
 
-  const buttonClass = variant ? secondaryClass : primaryClass;
+  let className = variant ? secondaryClass : primaryClass;
+
+  if (props.className) className = className + ' ' + props.className;
```

Comment:
Humm, not sure about that! This is tricky because we don't know in the API signature what className we are overriding. 🤔 

There's many others className in the component implementation.